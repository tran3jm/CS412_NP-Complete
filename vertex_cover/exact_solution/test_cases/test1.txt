14
0 1 2 3
1 0
2 0
3 0 4
4 3 5 6 7
5 4 8
6 4
7 4
8 5 9
9 8 10
10 9 11 12
11 10 13
12 10 13
13 11 12