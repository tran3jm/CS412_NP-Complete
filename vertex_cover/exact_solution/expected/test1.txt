Min number: 5
Vertices: 0 4 8 10 13